src/commands/deploy/command.ts

```diff
@@ -19,12 +20,17 @@ export async function deployCommand(options: GenezioDeployOptions) {
             debugLogger.debug("Deploying Next.js app");
             await nextJsDeploy(options);
             break;
+        case DeployType.NitroJS:
```

Comment:
The Nitro branding does not end in `.js` like Next.js or Express.js.

I would remove the `js` or `.js` suffixes from the codebase to maintain consistency with their documentation/landing page etc.